.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements/local.txt
+
+    - name: Run tests
+      run: |
+        python manage.py test
+
+    - name: Run codecov
+      uses: codecov/codecov-action@v1
```

Comment:
Sweep: there is already a ci.yml file no need to create another file. Add the codecov action to it.